Index.html

```diff
@@ -2,42 +2,82 @@
+                
+            <div class="even_space">
+                <div>
+                    <img src="./images/transformers200.jpg" height="350" width="300">
+                    <h1 class="center_text"> This Movie</h1>
+                </div>
+    
+                <div>
+                    <img src="./images/AOEPoster.jpg" height="350" width="300" onClick="alert('ALSO GOOD CHOICE')">
+                    <h1 class="center_text"> Other title</h1>
+                </div>
+                <!-- You can stick images in this DIV and they will auto space : )))))) Just make sure to put them in their own, tinier div -->
+            </div>
+
             <div class="even_space">
-                <div><img src="./images/transformers200.jpg" height="350" width="300">
-                <h1 class="center_text"> This Movie</h1></div>
                 <div>
-                <img src="./images/AOEPoster.jpg" height="350" width="300" onClick="alert('ALSO GOOD CHOICE')">
-                <h1 class="center_text"> Other title</h1>
+                    <img src="./images/transformers200.jpg" height="350" width="300">
+                    <h1 class="center_text"> This Movie</h1>
+                </div>
+    
+                <div>
+                    <img src="./images/AOEPoster.jpg" height="350" width="300" onClick="alert('ALSO GOOD CHOICE')">
+                    <h1 class="center_text"> Other title</h1>
                 </div>
                 <!-- You can stick images in this DIV and they will auto space : )))))) Just make sure to put them in their own, tinier div -->
+            </div>
 
+                        <div class="even_space">
+                <div>
+                    <img src="./images/transformers200.jpg" height="350" width="300">
+                    <h1 class="center_text"> This Movie</h1>
+                </div>
+    
+                <div>
+                    <img src="./images/AOEPoster.jpg" height="350" width="300" onClick="alert('ALSO GOOD CHOICE')">
+                    <h1 class="center_text"> Other title</h1>
+                </div>
+                <!-- You can stick images in this DIV and they will auto space : )))))) Just make sure to put them in their own, tinier div -->
             </div>
-        <input type="button" value="Go see the movies" onclick="alert('Go watch transformers')">   
-        <h2 style="display: inline-block" >Text 1</h2>
-        <h3 style="display: inline-block" >Text 2</h3>
-        <!--- Wanted to throw something in to see how updating code via gitbash would work - Vince-->
+    
+            <input type="button" value="Go see the movies" onclick="alert('Go watch transformers')">   
+            <h2 style="display: inline-block">Text 1</h2>
+            <h3 style="display: inline-block">Text 2</h3>
+            <!--- Wanted to throw something in to see how updating code via gitbash would work - Vince-->
+        </div>
+
+        <script type=text/javascript src="js/loginButtonLogic.js"></script>
```

Comment:
it's good practice to keep all your `script` tags at the very bottom of your HTML file, right before your last `</body>` tag.

that way, if your JS needs to reference your HTML file (like, for example, using `document.querySelector` to pull HTML elements), you won't get any `undefined` errors. your JS file needs to look "above" it to find the HTML.

you can move this `script` tag back into the `<head>` tag, then try to run your site. you'll see errors start to explode in your console.